### PR TITLE
DOCSP-38285 Beta Program Landing Page

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION 
- Create a "Beta Program" TOC drawer under "Reference" 
- Creates a Beta program landing page 

**QUESTION FOR REVIEWER(S)**: Is there anything else that a user would potentially want to know about the beta program that I may be forgetting?

## STAGING 
- [Beta Program Landing Page](https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCSP-38285-beta-program-landing/reference/beta-program/)
- [1.8 Release Notes](https://preview-mongodbajhuhmdb.gatsbyjs.io/cluster-sync/DOCSP-38285-beta-program-landing/release-notes/1.8/#cluster-to-cluster-sync-beta-program)

## JIRA 
https://jira.mongodb.org/browse/DOCSP-38285

## BUILD
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=66562f6bcf5bec1c295fc7d2

### CODE REVIEW

File: source/reference/beta-program/many-to-one.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.. _c2c-beta-many-to-one:
+
+===========
+Many To One
```
Comment (jmd-mongo) [inline source/reference/beta-program/many-to-one.txt:4]: [question]

Should this be hyphenated like "Many-To-One"?

File: source/reference/beta-program.txt [hunk 2]
```diff
@@ -12,13 +12,70 @@
    :depth: 2
    :class: singlecol
 
-Content for this page will be created in DOCSP-38285.
+.. include:: /includes/beta-program-intro.rst
+
+Each ``mongosync`` release will have a corresponding {+c2c-beta-program-short+}
```
Comment (jmd-mongo) [inline source/reference/beta-program.txt:17]: [suggestion]

Consider using the present tense here. Spitballing: "Each ``mongosync`` release has a corresponding...".

File: source/reference/beta-program.txt [hunk 3]
```diff
@@ -12,13 +12,70 @@
...
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 30 70
+
+   * - Feature
+     - Description
+
+   * - :ref:`c2c-beta-namespace-remapping`
+     - # DESCRIPTION TBD
```
Comment (jmd-mongo) [inline source/reference/beta-program.txt:60]: [question]

Should these descriptions still be TBD? Thanks!

**Review by jmd-mongo — COMMENTED** [review]
Thanks for these updates, @ajhuh-mdb! I've left some questions and a suggestion for your consideration.

Thanks!
Joe

File: source/reference/beta-program.txt (hunk 3 above)
Comment (ajhuh-mdb) [reply source/reference/beta-program.txt:60]: Yes! The corresponding tickets for the ones that are TBD haven't been completed yet. We'll be doing one more sweep through on [DOCSP-39992](https://jira.mongodb.org/browse/DOCSP-39992) to update any missing descriptions and make sure everything's covered.

File: source/reference/beta-program/many-to-one.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/reference/beta-program/many-to-one.txt:4]: Opting to update the title to be more specific.

**Pull request merged**